packages/aws-rfdk/lib/deadline/scripts/bash/enableCacheFilesd.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# Script to install and configure cachefilesd onto a Linux host.
+# This script is presently narrowly written for use on the ECS container host
+# instance(s) that are running the Deadline RCS containers in the RenderQueue construct.
+
+set -xeu
+
+if ! test -f /sbin/cachefilesd
+then
+  # The yum install can fail if there is no route to the yum repository.
+  # Since this is an AL2 instance, that is equivalent to requiring a route to
+  # the regional S3 endpoint on port 80.
+  sudo yum install -y cachefilesd || ( echo 'ERROR -- Failed to install cachefilesd' && exit 0 )
+
+  cat << EOF | sudo tee -a /etc/cachefilesd.conf
+# Allow 16k cull table entries
+culltable 14
+EOF
```

Comment:
Maybe I'm missing something, but should we apply this configuration regardless of whether `cachefilesd` was present (move outside the `if` block)?

Reply:
I was taking the approach that if it's already installed, then it has been configured by someone and I shouldn't mess with it. It's not normally installed on an ECS container host...